### ISSUE TITLE
Reflecting all the liked notes in the likes notes section.

### ISSUE THREAD

**Issue by Yashshukla11** [Yashshukla11/ExamTime_android#3]
Do proper statement management using bloc. All the liked notes will be reflected in the liked notes section of the app.

![WhatsApp Image 2024-05-03 at 3 35 54 PM](https://github.com/Yashshukla11/ExamTime_android/assets/112846526/d1cc4781-cbc2-4659-92ad-ae61ef89bd8a)

**Comment by Vardhan-ysh** [comment]
Hi, I would like to work on this.

**Comment by Saumya-28** [comment]
Hi @Yashshukla11 !
I would like to work on this issue, as I have previously worked with Bloc. Could you please assign this to me?

**Comment by Subhash23jan** [comment]
> Do proper statement management using bloc. All the liked notes will be reflected in the liked notes section of the app.
> 
> ![WhatsApp Image 2024-05-03 at 3 35 54 PM](https://private-user-images.githubusercontent.com/112846526/327756276-d1cc4781-cbc2-4659-92ad-ae61ef89bd8a.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzMTY0MTQsIm5iZiI6MTcxNTMxNjExNCwicGF0aCI6Ii8xMTI4NDY1MjYvMzI3NzU2Mjc2LWQxY2M0NzgxLWNiYzItNDY1OS05MmFkLWFlNjFlZjg5YmQ4YS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTBUMDQ0MTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODY3MTg5NzBiZDY2NTZjZTQyNzViMjBmYTA4ZDE3MGMyMmQzZTU0NDc1ZmE3ZjgzNTg2NGZiMzAyM2Q1OWYyNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.XF2-UKObYnAsAspl3TYjLxw0lQcIL6Hzw326MXJwowU)

@Yashshukla11 I would like to work on this feature 
and i have experience in flutter ,could you assign this for me?

**Comment by GauravDesale08** [comment]
Hello, I'm interested in contributing to this project. As i have good knowledge of bloc and other state managment. Please assign this to me.

**Comment by anvishelar** [comment]
I am Anvi Shelar. Under GSSOC'24, I would like to work on this issue. Pls assign me this!!

**Comment by vamzx** [comment]
Hello, I am Vamika, Contributor in GSSoC '24, I would like to work on this issue

**Comment by igopi77** [comment]
Hello @Yashshukla11 ,

My name is Gopinath, and I have a better knowledge in Bloc Management. If you're interested, you can view my work at the following repositories:

1. Helping Hand: [GitHub](https://github.com/igopi77/Helping-_Hands)
2. Stock Management: [GitHub](https://github.com/igopi77/Akshaya_Strore_V1)
3. Job Portal: [GitHub](https://github.com/MuthuvelA/JOBPORTAL_FRONTEND)

I am confident in my ability to make a significant contribution to your project. In particular, I believe I can successfully address the issue of "Reflecting all the liked notes in the likes notes section" within the ExamTime_Android repository.

For more insight into my professional background and projects, please visit my profiles:

LinkedIn: [Profile](https://www.linkedin.com/in/gopinath-siva-191a27259/)
GitHub: [Profile](https://github.com/igopi77)

I am eager to explore the possibility of collaboration and engaging in meaningful discussions.

Warm regards,
Gopinath

**Comment by Shilpajha29** [comment]
kindly assign this task to me

**Comment by AnanyaSinghal23** [comment]
I would like to work on this issue, please assign it to me. Thank you.

**Comment by Yashshukla11** [comment]
@Gaurav-Kushwaha-1225 @igopi77 any updates

**Comment by Gaurav-Kushwaha-1225** [comment]
@Yashshukla11 Can u give a sample login credentials for the developer version...
Every time I am getting an 'Invalid Credentials' issue.

**Comment by igopi77** [comment]
@Gaurav-Kushwaha-1225  Its working for me. Username : "demo@mail.xyz" , password : "Demodemo#1".

**Comment by igopi77** [comment]
@Yashshukla11 Sir, Iam working on it.

**Comment by Yashshukla11** [comment]
> @Yashshukla11 Sir, Iam working on it.

great, if there is any problem then DM me on Discord.

**Comment by igopi77** [comment]
@Yashshukla11 Sir, Did you written a backend for Liked notes. Iam trying to convey that, If we click a like button , the post will store on LIKED collection in backend.

Or I want to do it statically.

**Comment by igopi77** [comment]
@Yashshukla11 I have seen you Backend code. In that we have only store the notes id. So I have done this work by dynamically in FrontEnd.

**Comment by Yashshukla11** [comment]
> @Yashshukla11 I have seen you Backend code. In that we have only store the notes id. So I have done this work by dynamically in FrontEnd.

@igopi77 have you made PR

**Comment by Subhash23jan** [comment]
@Yashshukla11 i would like to work on this
could you assign this to me?

**Comment by igopi77** [comment]
> > @Yashshukla11 I have seen you Backend code. In that we have only store the notes id. So I have done this work by dynamically in FrontEnd.
> 
> @igopi77 have you made PR

@Yashshukla11 No, But I have completed this dynamically in frontend. If it's okay I can raise PR

**Comment by igopi77** [comment]
@Yashshukla11 sir, I have completed this issue. You can plese verify it and merge it sir.
Thank you.
